f2/src/forms/HowDidItStartForm.js

```diff
@@ -144,90 +143,50 @@ export const HowDidItStartForm = props => {
 
-            <Stack>
-              <P fontWeight="bold">
-                <Trans id="whenDidItStart.label" />
-              </P>
-              <P fontSize="md">
-                <Trans id="whenDidItStart.labelExample" />
-              </P>
-            </Stack>
-
-            <Stack flexDirection="row">
-              <Field name="startDay">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="startDay">
-                      <Trans id="whenDidItStart.startDay" />
-                      <TextInput
-                        id="startDay"
-                        name={props.input.name}
-                        value={props.input.value}
-                        onChange={props.input.onChange}
-                        w={70}
-                        h={36}
-                        mt={2}
-                        maxLength="2"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-              <Field name="startMonth">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="startMonth">
-                      <Trans id="whenDidItStart.startMonth" />
-                      <TextInput
-                        id="startMonth"
-                        name={props.input.name}
-                        value={props.input.value}
-                        onChange={props.input.onChange}
-                        w={70}
-                        h={36}
-                        mt={2}
-                        maxLength="2"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-              <Field name="startYear">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="startYear">
-                      <Trans id="whenDidItStart.startYear" />
-                      <TextInput
-                        id="startYear"
-                        name={props.input.name}
-                        value={props.input.value}
-                        onChange={props.input.onChange}
-                        w={110}
-                        h={36}
-                        mt={2}
-                        maxLength="4"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-            </Stack>
+            <FormArrayControl
+              label={<Trans id="whenDidItStart.label" />}
+              helperText={<Trans id="whenDidItStart.labelExample" />}
+            >
+              <Stack direction="row" spacing="2">
+                <Field
+                  name="startDay"
+                  label={<Trans id="whenDidItStart.startDay" />}
+                  component={TextInput}
+                  w={70}
+                  maxLength="2"
+                />
```

Comment:
We can use our Field component this handy way. Just with a few attributes, it renders all the appropriate markup and final-form components